tests/unit/models/conversation-test.js

```diff
@@ -1,18 +1,27 @@
 import { moduleForModel, test } from 'ember-qunit';
 import { testForAttributes } from 'code-corps-ember/tests/helpers/attributes';
 import { testForBelongsTo } from 'code-corps-ember/tests/helpers/relationship';
+import { run } from '@ember/runloop';
 
 moduleForModel('conversation', 'Unit | Model | conversation', {
-  // Specify the other units that are required for this test.
   needs: [
     'model:message',
+    'model:conversation',
+    'model:conversation-part',
     'model:user'
   ]
 });
 
-test('it exists', function(assert) {
-  let model = this.subject();
-  assert.ok(!!model);
+test('sortedConversationParts sorts insertedAt by asc', function(assert) {
+  let firstPart, secondPart;
+  run(() => {
+    firstPart = this.store().createRecord('conversation-part', { body: 'wat', insertedAt: new Date('2017-10-07') });
+    secondPart = this.store().createRecord('conversation-part', { body: 'foo', insertedAt: new Date('2017-10-06') });
+  });
+  let model = this.subject({ conversationParts: [firstPart, secondPart] });
+  assert.equal(model.get('sortedConversationParts.length'), 2);
+  assert.equal(model.get('sortedConversationParts').get('firstObject').get('body'), 'foo');
+  assert.equal(model.get('sortedConversationParts').get('lastObject').get('body'), 'wat');
```

Comment:
I would maybe consider renaming `firstPart` to `laterPart` and `secondPart` to `earlierPart`, then in the assertions you can use those variables to test against (rather than the explicit string). I think that could make it more obvious what's under test.

Reply:
It looks like you did the first part I suggested previously but not the second. Is there a reason for that?

Reply:
Ah.  I didn't see that.  

So What we are testing is that the `firstObject` is 'foo'.  I could add a test failure description to those.